lessons/linuxadmin/installation/virtualbox.md

```diff
@@ -39,8 +39,8 @@ Klikni na tlačitko „Nový“ a pokračuj podle obrázkového návodu:
 
 
 5. Tím je virtuální počítač vytvořen.
-   Spusť ho zelenou šipečkou „spustit“.
-   VirtualBox se zeptá na *obraz disku* ze kterého bude instalovat.
+   Spusť ho zelenou šipečkou „Spustit“.
```

Comment:
```suggestion
   Spusť ho zelenou šipečkou "Spustit".
```
jinde máme tyto uvozovky pokud se jedná na klikací tlačítko, tak bychom to měli mít stejně.